.github/workflows/dropbox_clear_space.yml

```diff
@@ -0,0 +1,39 @@
+name: Dropbox clear space
+
+on:
+  workflow_dispatch:
+  push: # this allows testing, can be removed when on master when it will allow for manual triggers
+  # schedule:
+  #   - cron:  '0 6 * * *'
+
+jobs:
+  dropbox_clear_space:
+    name: Clean dropbox action
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Python dependencies
+        working-directory: ./pipeline
+        run: |
+          pip install -r requirements.txt
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
```

Comment:
Is this doing anything?
it might be how luigi is magicly installed without being in the requirements...
However I don't see code like 
` if: steps.cache-primes.outputs.cache-hit != 'true'
      run: /generate-primes.sh -d prime-numbers`

Reply:
yeah, I'm not sure either. I'll look into this.